Examples/SubscriberExample/Sources/Screens/Map/MapViewController.swift

```diff
@@ -224,16 +249,24 @@ extension MapViewController: MKMapViewDelegate {
         scheduleResolutionUpdate()
     }
     
-    private func createAnnotationView(for annotation: TruckAnnotation) -> MKAnnotationView {
-        let annotationView = getAnnotationView(for: annotation)
+    private func createTruckAnnotationView(for annotation: TruckAnnotation) -> TruckAnnotationView {
+        let annotationView: TruckAnnotationView = getAnnotationView(for: annotation)
         annotationView.bearing = annotation.bearing
-        annotationView.backgroundColor = UIColor.blue.withAlphaComponent(0.7)
+        annotationView.zPriority = .init(rawValue: 1.0)
```

Comment:
I wonder whether it would be worth extracting this magic number to a constant called `FOREGROUND_Z` and then do the same for the `.zero` below and call it `BACKGROUND_Z`, WDYT? :thinking:

Reply:
Since we have only `foreground` and `background` views we can use predefined values `min` and `max`. Changed here 711666eb8cccea39fd8af90f2bd917db398acd3a

Reply:
The `min/max` approach is a change for the better but I still think that it would be even better to use the `foreground/backround` one :wink: The reason is that to understand the `min/max` you need to know if max priority means foreground and min means background or is it the other way around :sweat_smile: WDYT? :wink:

Reply:
you're right, changed here 41ce76c6221a73b907ff59ebb61ffcb1ba344c6e